components/Search/RefinementList.tsx

```diff
@@ -43,6 +43,7 @@ const StyledLabel = styled.label`
 const LabelName = styled.span`
   white-space: nowrap;
   overflow-x: hidden;
+  line-height: 1.2rem;
```

Comment:
We need to add the line height on both the text and number, the number is a little bit off now
![Skjermbilde 2023-03-27 kl  12 45 27](https://user-images.githubusercontent.com/1568252/227923587-3a18b9d0-8287-4bbf-943b-34a5f5c4f96f.png)